registry/super_editor.test

```diff
@@ -0,0 +1,38 @@
+# Tests for the super_editor mono repo.
+
+# Contacts
+contact=flutterbountyhunters@gmail.com
+
+# Update
+update=attributed_text/
+update=super_text_layout/
+update=super_editor/
+
+# Fetch the super_editor mono repo.
+fetch=git clone https://github.com/superlistapp/super_editor.git tests
+fetch=git -C tests checkout 004a6b93a56f18ec5320d340f61f8158febd59f8
+
+# Run the tests.
+test.posix=flutter_test_repo_attributed_text_test.sh
+test.windows=flutter_test_repo_attributed_text_test.bat
+
+test.posix=flutter_test_repo_super_text_layout_test.sh
+test.windows=flutter_test_repo_super_text_layout_test.bat
+
+test.posix=flutter_test_repo_super_editor_test.sh
+test.windows=flutter_test_repo_super_editor_test.bat
+
+# To test your tests, check out the flutter/flutter repository and
+# then, from the root of that repository, run:
+#
+#   bin/flutter; cd dev/customer_testing; flutter pub get; time ../../bin/cache/dart-sdk/bin/dart run_tests.dart --repeat=100 <path>
+#
+# ...where <path> is the path to this file.
+#
+# This should run with no output, without failing, and should in total
+# take less than 15 minutes (a hundred runs of your tests taking just
+# a few seconds each). If your tests take longer, mention this in your
+# PR and we will evaluate them to see how valuable they are (e.g. how
+# unique and different they are compared to other tests we already
+# have). The more valuable the tests, the more likely we are to accept
+# them despite them taking a long time to run.
```

Comment:
you may remove this comment also if you like (up to you, if you want it for your own purposes feel free to keep it)

Reply:
I'll leave it in to remind me